sdk/documentintelligence/Azure.AI.DocumentIntelligence/README.md

```diff
@@ -5,7 +5,6 @@
 Azure AI Document Intelligence is a cloud service that uses machine learning to analyze text and structured data from your documents. It includes the following main features:
 
 - Layout - Extract text, selection marks, table structures, styles, and paragraphs, along with their bounding region coordinates from documents.
-- General document - Analyze key-value pairs in addition to general layout from documents.
```

Comment:
General document model has been deprecated and is not supported by the service anymore.